Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.16-alpine AS build
+
+ADD . /go/src/timetracker
+WORKDIR /go/src/timetracker/cmd
+RUN CGO_ENABLED=0 go build -o /bin/timetracker
```

Comment:
It might be a good idea to add a `go test` step, mightn't it–just in case you accidentally built an image that didn't work!